pom.xml

```diff
@@ -17,8 +17,9 @@
     </modules>
 
     <properties>
-        <openMRSVersion>1.9.9</openMRSVersion>
+        <openMRSVersion>2.3.0</openMRSVersion>
         <heightweighttrackerVersion>1.0.3</heightweighttrackerVersion>
+        <reportingcompatibilityVersion>2.0.7-SNAPSHOT</reportingcompatibilityVersion>
```

Comment:
Is there any reason why you are using a snapshot instead of released version of the module?

Reply:
[Fixed](https://github.com/PIH/openmrs-module-rwandahivflowsheet/pull/2/commits/301ca92371f8e11a20b9ba38d8d43770645c14a8)
Seems 2.0.6 is the latest release.